support-frontend/assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -167,6 +173,48 @@ function campaignSpecificDetails() {
   return {};
 }
 
+function goalReachedTemplate() {
+  if (isFrontlineCampaign()) {
```

Comment:
I think this condition should be an `&&` with `props.tickerGoalReached` - otherwise in principle we could show a blank page if the ticker goal was reached but it wasn't the campaign page (though I don't think this would do anything in practice right now)

Reply:
actually - it's probably fine the way it is